Libraries/Drei/views.lisp

```diff
@@ -73,7 +73,9 @@ If empty, tabs every TAB-WIDTH are assumed.")))
                    (recorded-stream recorded-stream)) tabify
     (unless (eq stream recorded-stream)
       ;; Update the recorded values.
-      (setf space-width (stream-character-width stream #\Space)
+      (setf space-width (climb:text-style-character-width (medium-text-style stream)
+							  stream
+							  #\Space)
```

Comment:
please detabify the file (changes contain tabs). I've missed that part in the title-pane PR - you may want to tweak your editor settings. I believe that this is enough for emacs:
`(setq-default indent-tabs-mode nil)`

Reply:
done.